### ISSUE TITLE
How to sort list items using Filter?

### ISSUE THREAD

**Issue by robinisaksson** [umbraco/Umbraco.Headless.Client.NodeJs#21]
Hi,
When I use the "filter" function my results always seems to be ordered by "Last edited".

How can I control the order of my filtered results? I would like it to respect the "sortOrder".

```
client.delivery.content.filter(body, options)
// FAIL: items are not sorted by "sortOrder"

client.delivery.content.byContentType('type', options)
// SUCCESS: items are sorted by "sortOrder"
```

**Comment by sitereactor** [comment]
Good point. The order is not something we initially considered, but I can see the idea / benefit of being able to define how the results are ordered.

I'll add this as a consideration for our upcoming sprint.
